src/raspbian/12/helix/arm32v6/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM mcr.microsoft.com/dotnet-buildtools/prereqs:raspbian-10-crossdeps-local as builder
+FROM --platform=linux/arm/v7 mcr.microsoft.com/dotnet-buildtools/prereqs:raspbian-12-crossdeps-local as builder
```

Comment:
why are we passing --platform here?

Reply:
Should it be [`linux/arm/v6`](https://docs.docker.com/build/building/multi-platform/)?

Reply:
It shouldn't be necessary at all. That would only be necessary if you were pulling a multi-arch tag which isn't the case here.

Reply:
If we are going to replace "armv6" image and new image doesn't provide armv6, then it is not the right image? I think `uname -a` etc. should show `armv6l`. v7 and v6 are very different ISAs; which is why we have ARM and ARMV6 as [separate architectures](https://learn.microsoft.com/en-us/dotnet/api/microsoft.build.utilities.processorarchitecture?view=msbuild-17-netcore#fields).

Reply:
> Without `--platform` I am getting the following in ci:
> 
> ```
> Dockerfile:1
> --------------------
>    1 | >>> FROM mcr.microsoft.com/dotnet-buildtools/prereqs:raspbian-12-crossdeps-local as builder
>    2 |     
>    3 |     ARG ROOTFS_DIR=/crossrootfs
> --------------------
> ERROR: failed to solve: mcr.microsoft.com/dotnet-buildtools/prereqs:raspbian-12-crossdeps-local: mcr.microsoft.com/dotnet-buildtools/prereqs:raspbian-12-crossdeps-local: not found
>    at Microsoft.DotNet.ImageBuilder.ExecuteHelper.Execute(ProcessStartInfo info, Func`2 executor, Boolean isDryRun, String errorMessage, String executeMessageOverride) in /image-builder/src/ExecuteHelper.cs:line 101
>    at Microsoft.DotNet.ImageBuilder.ExecuteHelper.ExecuteWithRetry(ProcessStartInfo info, Action`1 processStartedCallback, Boolean isDryRun, String errorMessage, String executeMessageOverride) in /image-builder/src/ExecuteHelper.cs:line 60
>    at Microsoft.DotNet.ImageBuilder.ExecuteHelper.ExecuteWithRetry(String fileName, String args, Boolean isDryRun, String errorMessage, String executeMessageOverride) in /image-builder/src/ExecuteHelper.cs:line 45
>    at Microsoft.DotNet.ImageBuilder.DockerService.BuildImage(String dockerfilePath, String buildContextPath, String platform, IEnumerable`1 tags, IDictionary`2 buildArgs, Boolean isRetryEnabled, Boolean isDryRun) in /image-builder/src/DockerService.cs:line 60
>    at Microsoft.DotNet.ImageBuilder.DockerServiceCache.BuildImage(String dockerfilePath, String buildContextPath, String platform, IEnumerable`1 tags, IDictionary`2 buildArgs, Boolean isRetryEnabled, Boolean isDryRun) in /image-builder/src/DockerServiceCache.cs:line 37
>    at Microsoft.DotNet.ImageBuilder.Commands.BuildCommand.BuildImage(PlatformInfo platform, IEnumerable`1 allTags) in /image-builder/src/Commands/BuildCommand.cs:line 483
> ```

This doesn't make sense to me. This happens on the CI in AzDO? I can't find a failed build with that error. I'd like to see the build log to understand what's happening. Getting a `not found` error would mean the image tag doesn't exist at all, regardless of its platform.

Reply:
Ok, I get why `--platform` is needed now. That's fine. I'll leave it to the other reviewers to determine whether consuming a v7 image from a v6 image is the right thing to do.